gen/mailman

```diff
@@ -65,7 +64,7 @@ for my $listName (keys %$mailinglistStruc) {
     # As a first line, print header
```

Comment:
I'm just wondering if we should sort also the lists themselves, so that if generating one of them fails, we can reproduce also the order in which they are generated. But its probably not needed.